doc/modules/ROOT/pages/usage/server.adoc

```diff
@@ -187,6 +216,18 @@ is an epic security hole! As the connections are insecure (no
 authentication, no authorization) by default anyone can connect to
 your app and modify its behaviour or run code on the remote host.
 
+You can also ask nREPL to listen on a UNIX domain (filesystem) socket
```

Comment:
Probably we should also add a note that this won't work on Windows. :-)

Reply:
Hmm, no idea - though junixsocket claims to provide some support.  Suppose someone with a system handy could try it there: https://github.com/kohlschutter/junixsocket#junixsocket